.github/workflows/ci.yaml

```diff
@@ -25,4 +25,4 @@ jobs:
 #      - name: integration test
```

Comment:
I had thought perhaps I might use a shorter test for PRs. But I'm thinking now I probably won't ever do that. Might make sense to just delete these unused steps (which still have references to the old profile).

Reply:
3min is fine. Locally, just don't run the invoker stuff, and you are done.
Thinking of invoker: I wanted to raise another issue where I put everything into a profile instead (and making 'install' not needed for integration testing). install will clutter and spam your local .m2 folder, so it is common to avoid this whenever possible.
But that's for another issue. I can put all the details there.

Reply:
I removed that line in this PR: https://github.com/groovy/GMavenPlus/pull/268